packages/@react-spectrum/accordion/chromatic/Accordion.chromatic.tsx

```diff
@@ -22,7 +22,7 @@ const meta: Meta<SpectrumAccordionProps<object>> = {
 
 export default meta;
 
-const Template = (args) => (
+export const Template = (args) => (
```

Comment:
i think this has be excluded from stories in the meta? otherwise it'll try to make a story for it?

Reply:
Good catch, I added `excludeStories` to the placed that I added `export`